PowerEditor/resources/nppPluginList.json

```diff
@@ -15,7 +15,7 @@
 			"folder-name": "mime-tools",
 			"display-name": "Mime tools",
 			"repository": "https://github/npp_plugins/mimetools/",
-			"description": "MIME plugin for Notepad++ implements several main functionnalities defined in MIME (Multipurpose Internet Mail Extensions).",
```

Comment:
Already correct at https://github.com/notepad-plus-plus/nppPluginList/blob/master/src/pl.x86.json

Reply:
@chcg Do we need this file in npp repository? I don't think so.

Reply:
@SinghRajenM You are right. This file is not needed any longer in the npp repo itself and is outdated version 1.0 anyway.

Reply:
@SinghRajenM Yes, this file should be removed.